easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.4-2023a.yml

```diff
@@ -5,3 +5,7 @@ easyconfigs:
         from-commit: 6cbfbd7d7a55dc7243f46d0beea510278f4718df
         # see https://github.com/easybuilders/easybuild-easyblocks/pull/3467
         include-easyblocks-from-commit: c3aebe1f133d064a228c5d6c282e898b83d74601
+  - waLBerla-6.1-foss-2023a-CUDA-12.1.1.eb:
```

Comment:
@Neves-P This should be listed in an easystack under `accel`

Reply:
Addressed in 69e1ba5